### DISCUSSION TITLE
Root password reset?

### DISCUSSION THREAD

**Discussion by Wiizoi** [Joshua-Riek/ubuntu-rockchip#379] [Q&A]
Hello, I'm new to linux and I've forgotten the root password. I've tried getting into the grub menu to try and reset but it doesn't seem to work. Is there any way to reset the root password?

**Comment by gerardbita** [comment]
I have the same problem here

**Comment by Joshua-Riek** [comment]
You guys can boot another Linux operating system and mount the filesystem that contains your os with a lost root password. Simply chroot into the mounted partition, and you can change the password with `passwd` command.

https://serverfault.com/questions/970110/ubuntu-root-password-reset

**Reply by WINTX** [reply]
Do you mean, in my case, taking the microSD and putting it in the Ubuntu that I have installed on my PC, for example?

**Reply by Joshua-Riek** [reply]
Yes
